### PR TITLE
output scores in detection layer

### PR DESCRIPTION
**Pull request by jhung0**

### CODE REVIEW

File: keras_rcnn/layers/object_detection/_anchor_target.py [hunk 1]
```diff
@@ -268,12 +269,12 @@ def unmap(data, count, inds_inside, fill=0):
     size count) """
 
     if keras.backend.ndim(data) == 1:
-        ret = keras.backend.ones((count,), dtype=keras.backend.floatx()) * fill
+        ret = tensorflow.ones((count,), dtype=keras.backend.floatx()) * fill
```
Comment (0x00b1) [inline keras_rcnn/layers/object_detection/_anchor_target.py]: Is there an issue with `keras.backend.ones`?

File: keras_rcnn/layers/object_detection/_anchor_target.py [hunk 2]
```diff
@@ -268,12 +269,12 @@ def unmap(data, count, inds_inside, fill=0):
...
 
         inds_nd = keras.backend.expand_dims(inds_inside)
     else:
-        ret = (count,) + keras.backend.int_shape(data)[1:]
-        ret = keras.backend.ones(ret, dtype=keras.backend.floatx()) * fill
+        ret = (count, keras.backend.shape(data)[1])
+        ret = tensorflow.ones(ret, dtype=keras.backend.floatx()) * fill
```
Comment (0x00b1) [inline keras_rcnn/layers/object_detection/_anchor_target.py]: Is there an issue with `keras.backend.ones`?

File: tests/preprocessing/test_object_detection.py [hunk 3]
```diff
@@ -26,7 +26,7 @@ def test_scale_shape():
 class TestObjectDetectionGenerator:
     def test_flow(self):
 
-        image = keras.layers.Input((448, 448, 3))
+        image = keras.layers.Input((None, None, 3))
```
Comment (0x00b1) [inline tests/preprocessing/test_object_detection.py]: Shape inference is working? 😮

Comment (jhung0) [reply tests/preprocessing/test_object_detection.py]: Yes seems to work

File: keras_rcnn/layers/object_detection/_anchor_target.py (hunk 2 above)
Comment (jhung0) [reply keras_rcnn/layers/object_detection/_anchor_target.py]: ```tensorflow.ones``` is somewhat different than ```keras.backend.ones```

Comment (jhung0) [reply keras_rcnn/layers/object_detection/_anchor_target.py]: I'm not sure why they're different but ```keras.backend.ones``` can't handle when I try to define a tensor with shape given by ```keras.backend.shape```

**Pull request merged**